examples/resource-management/04-RoleAssignments.sh

```diff
@@ -3,25 +3,25 @@ set -e
 printf "\n=== Managing Role Assignments in Azure ===\n"
 
 printf "\n1. Creating a new resource group: %s and location: %s.\n" "$groupName" "$location"
-az group create --name "$groupName" --location "$location"
+az resourcemanager group create --name "$groupName" --location "$location"
 
 printf "\n2. Creating a new Role Assignment.\n"
 export MSYS_NO_PATHCONV=1
-users=$(az ad users get)
+users=$(az activedirectory users ls)
 userId=$(echo $users | cat | jq '.[0].Id' -s --raw-output)
 echo "UserID: $userId"
-roleDefinitions=$(az role definition get)
+roleDefinitions=$(az security role definition ls)
```

Comment:
Jason voted for keeping the security term to keep azure security concepts in a single location and be extra clear about what the role was